src/settings/global_user.rs

```diff
@@ -10,7 +10,8 @@ use config::{Config, Environment, File};
 #[derive(Clone, Debug, Deserialize, Serialize)]
 pub struct GlobalUser {
     pub email: String,
-    pub api_key: String,
+    pub api_key: Option<String>,
+    pub api_token: Option<String>,
```

Comment:
Maybe it would simplify the logic a bit:
```rust
pub enum Authentification { ApiToken(String), ApiKey(String) };
pub struct GlobalUser {
  pub email: String,
  pub authentification: Authentification,
}
```

Then in `src/http.rs` you could:

```rust
    match &user.authentification {
         ApiKey(key) => headers.insert("X-Auth-Key", HeaderValue::from_str(&key).unwrap()),
         ApiToken(token) => headers.insert(
             "Authorization",
             HeaderValue::from_str(&format!("Bearer {}", &token)).unwrap(),
         ),
     };
```

Reply:
Two points here:
1. email is only used with API key, so this enum doesn't need to pair an email with an api_token
1. we're going to remove API key support in an upcoming release anyways so I don't think we need to introduce an enum of two values when the second value is going to get removed in the near future.